cdap-api/src/main/java/co/cask/cdap/api/plugin/PluginConfig.java

```diff
@@ -37,4 +40,23 @@
   public final PluginProperties getProperties() {
     return properties;
   }
+
+  /**
+   * Returns true if property value is a macro; false otherwise.
+   * @param fieldName
+   * @return
+   */
+  public boolean isMacro(String fieldName) {
```

Comment:
it is more accurate for this to be called 'containsMacro', since only part of the field may be a macro.